src/wix/WixToolset.Core/Compiler_Package.cs

```diff
@@ -190,12 +190,6 @@ private void ParsePackageElement(XElement node)
                     Value = "Installation Database"
                 });
 
-                this.Core.AddSymbol(new SummaryInformationSymbol(sourceLineNumbers)
```

Comment:
`SummaryInformation` is an optional element. This default value still needs to be set when no `SummaryInformation` is provided.